tests/Microcks.Testcontainers.Tests/Async/Amqp/MicrocksAsyncAmqpFeatureTest.cs

```diff
@@ -0,0 +1,109 @@
+//
+// Copyright The Microcks Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License")
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+using Testcontainers.RabbitMq;
+using DotNet.Testcontainers.Networks;
+using DotNet.Testcontainers.Builders;
+using System;
+using System.Threading;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using Microcks.Testcontainers.Connection;
+
+namespace Microcks.Testcontainers.Tests.Async.Amqp;
+
+public class MicrocksAsyncAmqpFeatureTest : IAsyncLifetime
+{
+    private RabbitMqContainer _rabbitMqContainer;
+    private MicrocksContainerEnsemble _ensemble;
+    private INetwork _network;
+
+    public async Task InitializeAsync()
+    {
+        _network = new NetworkBuilder().Build();
+        _rabbitMqContainer = new RabbitMqBuilder()
+            .WithImage("rabbitmq:3.13-management-alpine")
```

Comment:
This image looks a bit old comparing to the one that is used for Java (see https://github.com/microcks/microcks-testcontainers-java/blob/main/src/test/java/io/github/microcks/testcontainers/MicrocksContainersEnsembleTest.java#L218)